pkg/collector/corechecks/snmp/checkconfig/config_validate_enrich.go

```diff
@@ -10,8 +10,15 @@ var validMetadataResources = map[string]map[string]bool{
 		"name":          true,
 		"description":   true,
 		"sys_object_id": true,
+		"location":      true,
```

Comment:
Why `ip_address` and `network_address` are missing here?

Reply:
The `ip_address` and `subnet` (do you mean `device.subnet` field?) are not based on profiles definition, so no need to validate them here and they can't be overwritten via profile.

- ip_address is based on instance ip_address
- subnet is from autodiscovery